pandas/io/packers.py

```diff
@@ -485,9 +486,14 @@ def decode(obj):
         def create_block(b):
             values = unconvert(b['values'], dtype_for(b['dtype']),
                                b['compress']).reshape(b['shape'])
+
+            if 'locs' in b:
```

Comment:
can you add a comment about this (e.g. that its compat for duplicates and should be used on-going), and put the issue number as a comment